frontend/src/LineGraph.res

```diff
@@ -3,6 +3,33 @@ type annotation
 type point
 type event
 
+let computeStdDev = (~mean, xs) => {
+  let var = ref(0.0)
+  let n = Array.length(xs)
+  for i in 0 to n - 1 {
+    let x = xs[i] -. mean
+    var := var.contents +. x *. x
+  }
+  sqrt(var.contents /. float_of_int(n - 1))
+}
```

Comment:
I was wondering if we should use a package that can compute the std deviation and mean for us? Or we'd rather not introduce more dependencies?